src/Uno.UI/LayoutProvider.Android.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.App;
+using Android.Graphics;
+using Android.Graphics.Drawables;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using Windows.Foundation;
+using Rect = Android.Graphics.Rect;
+
+namespace Uno.UI
+{
+	public class LayoutProvider
+	{
+		public delegate void LayoutChangedListener(Rect statusBar, Rect keyboard, Rect navigationBar);
+
+		public static LayoutProvider Instance { get; private set; }
```

Comment:
Ditch this property. There's already one pseudo-singleton in the form of ApplicationActivity.Instance, just access it via that.